changelogs/unreleased/gh-8416-space-and-index-names-in-iproto-requests.md

```diff
@@ -0,0 +1,10 @@
+## feature/box
+
+* Added support for accepting IPROTO requests with space or index name instead 
+  of identifier (gh-8146).
```

Comment:
BTW, it also closes #4479, I guess.

Reply:
I forgot about #4479, should I close #8146 as duplicate?

Reply:
I guess that the fresh issue is tracked in some epics/boards and I would just close both by one patch to not do any mess in the management tools.

Reply:
Closed #4479 as duplicate.